warehouse/query-core/src/main/java/datawave/query/index/lookup/CreateUidsIterator.java

```diff
@@ -80,17 +82,54 @@
     public void init(SortedKeyValueIterator<Key,Value> source, Map<String,String> options, IteratorEnvironment env) throws IOException {
         src = source;
         if (null != options) {
+            
             final String collapseOpt = options.get(COLLAPSE_UIDS);
-            if (null != collapseOpt) {
+            if (StringUtils.isNotBlank(collapseOpt)) {
```

Comment:
seems like we are repeating this parsing of the option code for a boolean and then an integer.  Perhaps refactor into separate methods?

Reply:
This way of parsing the argument will tell you if a non-boolean value was passed in. I think that's sufficient reason to keep it as is.